builder/scaleway/builder.go

```diff
@@ -33,7 +33,11 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 }
 
 func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packer.Artifact, error) {
-	client, _ := api.NewScalewayAPI(b.config.Organization, b.config.Token, b.config.UserAgent, b.config.Region)
+	client, err := api.NewScalewayAPI(b.config.Organization, b.config.Token, b.config.UserAgent, b.config.Region)
+
+	if err != nil {
+		return nil, err
```

Comment:
Add `ui.Error(err.Error())` before this.

Reply:
Fixed in 2cc13e74fb650d31f3fa64d2c23b1baf3cdeec7a